packages/expo-doctor/src/checks/DirectPackageInstallCheck.ts

```diff
@@ -11,6 +12,14 @@ const baseCheckItem = {
   sdkVersionRange: '*',
 };
 
+const expoFirebaseCheckItem = {
```

Comment:
looks like this snuck in the cocoapods pr!

Reply:
That was intentional: 
<img width="682" alt="image" src="https://github.com/expo/expo/assets/8053974/9293a14c-ba12-49ed-a785-b79690efb13f">
I can pull it out into its own thing if needed, but these checks are the closest we have to "add configuration, get check" right now, thought I could let them tag along.